src/core/server/integration_tests/elasticsearch/client.test.ts

```diff
@@ -71,15 +73,17 @@ function createFakeElasticsearchServer() {
 describe('fake elasticsearch', () => {
   let esServer: http.Server;
   let kibanaServer: Root;
-  let kibanaHttpServer: http.Server;
+  let esStatus$: ReplaySubject<ServiceStatus<ElasticsearchStatusMeta>>;
 
   beforeAll(async () => {
     kibanaServer = createRootWithCorePlugins({ status: { allowAnonymous: true } });
     esServer = createFakeElasticsearchServer();
 
-    const kibanaPreboot = await kibanaServer.preboot();
-    kibanaHttpServer = kibanaPreboot.http.server.listener; // Mind that we are using the prebootServer at this point because the migration gets hanging, while waiting for ES to be correct
-    await kibanaServer.setup();
```

Comment:
The status API is no longer returning the full status on the preboot handler, so I had to adapt this test to get the info from elsewhere.